Applications/Games/The Sims 2/Local/script.js

```diff
@@ -0,0 +1,39 @@
+const LocalScript = include("engines.wine.quick_script.local_installer_script");
+include("engines.wine.verbs.vcrun2010");
+include("engines.wine.verbs.vcrun2013");
+include("utils.functions.net.resource");
+include("utils.functions.filesystem.files");
+include("utils.functions.filesystem.extract");
+include("utils.functions.apps.resources");
+
+new LocalInstallerScript()
+    .name("The Sims 2")
+    .editor("Electronic Arts")
+    .applicationHomepage("http://thesims2.ea.com")
+    .author("ZemoScripter")
+    .category("Games")
+    .executable("Origin.exe")
+    .wineVersion("4.5")
+    .wineDistribution("staging")
+    .preInstall(function (wine) {
+        wine.vcrun2010();
+        wine.vcrun2013();
+        var registrySettings = new AppResource().application([TYPE_ID, CATEGORY_ID, APPLICATION_ID]).get("registry.reg");
+        wine.regedit().patch(registrySettings);
+     })
+    .postInstall(function (wine) {
+        var fixes = new Resource()
+            .wizard(this.wizard())
+            .url("https://raw.githubusercontent.com/tannisroot/installer-fixes/master/sims2_fixes.tar.xz")
+            .name("sims2_fixes.tar.xz")
+            .get();
+        var username = java.lang.System.getenv("USERNAME")
```

Comment:
Which username are you fetching here? I wouldn't recommend accessing system environment variables here

Reply:
https://www.tutorialspoint.com/java/lang/system_getenv_string.htm it fetches current user name

Reply:
`System.getenv` is very platform dependent. Who can say for sure that your target platform has a `USERNAME` environment variable? Try using `java.lang.System.getProperty("user.name")` instead.

Additionally please use `const System = Java.type("java.lang.System");` in the header and then use it with:

```javascript
const username = System.getProperty("user.name");
```